testing/src/main/java/com/yubico/yubikit/testing/piv/PivDeviceTests.java

```diff
@@ -22,7 +22,7 @@
 import com.yubico.yubikit.core.application.BadResponseException;
 import com.yubico.yubikit.core.smartcard.ApduException;
 import com.yubico.yubikit.core.smartcard.SW;
-import com.yubico.yubikit.piv.InvalidPinException;
+import com.yubico.yubikit.core.application.InvalidPinException;
```

Comment:
If we no longer catch the deprecated `piv` exception we should probably add an isinstance() check to make sure we're still throwing the deprecated type.

Reply:
good suggestion!

Reply:
I reverted the change of the exception class and instead suppress the warning in the test class. When we finally remove the deprecated class there will not be any warnings.